Framework/Core/include/Framework/AnalysisTask.h

```diff
@@ -246,18 +246,18 @@ struct AnalysisDataProcessorBuilder {
           if constexpr (soa::is_type_with_originals_v<std::decay_t<T>>) {
             using O = typename framework::pack_element_t<0, typename std::decay_t<G>::originals>;
             using groupingMetadata = typename aod::MetadataTrait<O>::metadata;
-            return std::string("f") + groupingMetadata::tableLabel() + "ID";
+            return std::string("fIndex") + groupingMetadata::tableLabel();
```

Comment:
I am not 100% sure this is enough... Don't we depend on ID to be there in a few more places? That said, can't we simply say that indices have "ID" at the end?

Reply:
It is in two places as far as I know. Column definition and here. Where else do you think? At least the code runs...
Regarding Index or ID: When we discussed in WP14, we said we would like to have fIndex in the front, so that's what I did now. It is forbidden through a compile time assert for other fields.